joss_qmc/paper.md

```diff
@@ -0,0 +1,114 @@
+---
+title: 'Quasi-Monte Carlo Methods in Python'
+tags:
+  - Python
+  - SciPy
+  - statistics
+  - Quasi-Monte Carlo methods
+authors:
+  - name: Pamphile T. Roy
+    affiliation: 1
+    corresponding: true
+    orcid: 0000-0001-9816-1416
+  - name: Art B. Owen
+    affiliation: 2
+    orcid: 0000-0001-5860-3945
+  - name: Maximilian Balandat
+    affiliation: 3
+    orcid: 0000-0002-8214-8935
+  - name: Matt Haberland
+    affiliation: 4
+    orcid: 0000-0003-4806-3601
+affiliations:
+    - name: Quansight
+      index: 1
+    - name: Stanford University
+      index: 2
+    - name: Meta
+      index: 3
+    - name: California Polytechnic State University, San Luis Obispo, USA
+      index: 4
+date: 30 August 2022
+bibliography: paper.bib
+
+---
+
+# Summary
+
+
+# Statement of need
+
+NumPy random number generators (`numpy.random`) have become the de-facto
```

Comment:
```suggestion
NumPy pseudorandom number generators (`numpy.random`) have become the de-facto
```
The intent is to help distinguish between "pseudorandom" as a class of algorithms for random number generation and NumPy's implementation of these algorithms. That way, it is clear that we are thanking NumPy for a solid implementation of classical pseudorandom numbers and only criticizing the shortcomings of pseudorandom numbers themselves.